lib/drivers/tandemDriver.js

```diff
@@ -1221,8 +1230,13 @@ module.exports = function (config) {
 
     function iterate(err, result) {
       if (err) {
-        debug('error retrieving record ', result);
-        callback(err, null);
+        if(err.name === 'multi-fail') {
```

Comment:
Yes, we're also using `err.name` in the Bayer driver. My understanding is that each `Error` [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name) contains both a `name` and `message` property.